apps/museums/fields/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   },
   'contact-phone': {
     mixin: 'input-text',
-    validate: ['required', 'phonenumber']
+    validate: 'required'
```

Comment:
Changing the formatter wasn't an option.

`s/formatter/validator`?

I'm inclined to agree, but it's buried right down in `hof-form-controller` so this was a pragmatic fix in the short/medium term.